kousa/lib/beef/notifications.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Beef.Notifications do
+  alias Beef.Schemas.Notification
+
+  def insert(data) do
+    %Notification{}
+    |> Notification.insert_changeset(data)
+    |> Beef.Repo.insert(on_conflict: :nothing, returning: true)
```

Comment:
@benawad  could you help me when u get the time? How can I load assoc directly when a record is inserted? searched a lot but couldn't figure it out :(

Reply:
just fetch the user by id in the business logic fn